src/CVObjectDetection.cpp

```diff
@@ -318,15 +318,15 @@ bool CVObjectDetection::SaveObjDetectedData(){
     pb_objdetect::ObjDetect objMessage;
 
     //Save class names in protobuf message
-    for(int i = 0; i<classNames.size(); i++){
+    for(size_t i = 0; i<classNames.size(); i++){
         std::string* className = objMessage.add_classnames();
         className->assign(classNames.at(i));
     }
 
     // Iterate over all frames data and save in protobuf message
     for(std::map<size_t,CVDetectionData>::iterator it=detectionsData.begin(); it!=detectionsData.end(); ++it){
         CVDetectionData dData = it->second;
-        pb_objdetect::Frame* pbFrameData;
+        // pb_objdetect::Frame* pbFrameData;
```

Comment:
Is this pointer used?

Reply:
AFAICT no, I think I commented it out rather than deleting it only to act as a placeholder/reminder to @BrennoCaldato (or anyone) that a `pbFrameData` **was** being allocated here, presumably for some reason (but then nothing was ever done with it).

Probably would've made sense to leave an actual **comment**, along with the commented-out code.